src/aspire/utils/preprocess.py

```diff
@@ -302,3 +303,34 @@ def vol2img(volume, rots, L=None, dtype=None):
         im = im * m_reshape(np.exp(2*np.pi*1j*(grid2d['x'] +grid2d['y'])/(2*lv)), (lv, lv, 1))
 
     return np.real(im)
+
+
+def fuzzy_mask(L, r0, risetime, origin=None):
```

Comment:
Does this have a unit test? If it doesn't it should for all the dimensions you need it to work for...

Reply:
I see you added a unit test, but it looks like only for one dimension.  Since the code actually had bugs that caused it to be broken for half it's use cases (3D), and maybe wasn't reproducing in 2D either based on your remarks, do you think it might be worth setting up both 2D and 3D while it is still fresh?

Reply:
We suppose to come back for the preprocess steps. But I have added the test in commit  03944b1.

Reply:
The test is for 2D case, the exact one we want. In the original Matlab, r0 is scale.

Reply:
k